lib/iris/fileformats/grib/_grib1_load_rules.py

```diff
@@ -14,28 +14,29 @@
 #
 # You should have received a copy of the GNU Lesser General Public License
 # along with Iris.  If not, see <http://www.gnu.org/licenses/>.
-
-from __future__ import (absolute_import, division, print_function)
-from six.moves import (filter, input, map, range, zip)  # noqa
+"""
+Module to support the loading and conversion of a GRIB1 message into
+cube metadata.
```

Comment:
I added this docstring. I'm not sure if it was necessary but all the other modules had a doc string so it seemed strange that this one didn't